src/dialog/marisa.c

```diff
@@ -115,6 +317,34 @@ DIALOG_TASK(marisa, Stage4PostBoss) {
 	VARIANT(kurumi, defeated);
 	FACE(kurumi, defeated);
 
+	FACE(marisa, smug);
+	MSG(marisa, "Whoever has the best petticoat wins. It’s a scientific fact.");
+	FACE(marisa, sweat_smile);
+	MSG(marisa, "… but since when do I care about scientific facts?");
+
+	MSG(kurumi, "W-wait, what was I doing again?");
+	MSG(kurumi, "(And didn’t I see this nerdy looking girl before…?)");
+
+	FACE(marisa, normal);
+	MSG(marisa, "Ah, sorry, sorry. I think ya slipped and hit your head.");
+	MSG(marisa, "I’m Gensōkyō’s… petticoat inspector.");
+	MSG(marisa, "I’m here to inspect the petticoats in this here mansion.");
+	FACE(marisa, happy);
+	MSG(marisa, "Y’were just lettin’ me in, so I’ll be off, then.");
```

Comment:
Another one of those "you" contractions I'm not a big fan of... maybe just use "ya"?

```suggestion
	MSG(marisa, "Ya were just lettin’ me in, so I’ll be off, then.");
```